src/main/java/apoc/graph/inverse/builder/IdBuilder.java

```diff
@@ -0,0 +1,53 @@
+package apoc.graph.inverse.builder;
+
+import apoc.graph.util.FromConfig;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static apoc.export.util.FormatUtils.formatString;
+
+public class IdBuilder {
```

Comment:
it's probably a little bit overkill
why do we need the fields map in addition to type and id fields?
should type be string or string array/collection?